packages/builders/src/components/selectMenu/MentionableSelectMenu.ts

```diff
@@ -91,7 +93,6 @@ export class MentionableSelectMenuBuilder extends BaseSelectMenuBuilder<APIMenti
 		>
```

Comment:
[Can't comment on the actual method name, but it's `addDefaultValues()`]

I believe we should remove this method (and potentially `setDefaultValues`, although that would remove the ability to clear the default values after adding them), since there are already separate ones for the two types of *things* you can add

Reply:
I'd rather keep it exactly for the ability to clear stuff.